tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -90,19 +92,39 @@ def _process_cancelled_event(self, calendar_event):
             todoist_item = self.todoist.get_item_by_id(todoist_id)
             if todoist_item is not None:
                 self.todoist.delete_item(todoist_item)
+    
+    def _process_updated_event(self, calendar_event):
+        print('Updating event|', calendar_event)
+        todoist_item = self.todoist.get_item_by_id(_todoist_id(calendar_event))
+        if todoist_item is None:
```

Comment:
it could happen if todoist item was deleted by the user. We should recreate it in that case.